mmdeploy/pytorch/functions/chunk.py

```diff
@@ -31,3 +32,19 @@ def chunk__ncnn(ctx, self, num_chunks: int, dim: int = 0) -> torch.Tensor:
     ]
 
     return output
+
+
+@FUNCTION_REWRITER.register_rewriter(
```

Comment:
Why do we need a chunk rewriter for torchscript?

Reply:
@grimoire It breaks as follow, maybe there's better solution?
```
  File "/home/PJLAB/maningsheng/projects/openmmlab/mmdeploy/mmdeploy/backend/torchscript/wrapper.py", line 49, in __init__
    self.ts_model = torch.jit.load(self.ts_model)
  File "/home/PJLAB/maningsheng/anaconda3/envs/pt1.8/lib/python3.7/site-packages/torch/jit/_serialization.py", line 161, in load
    cpp_module = torch._C.import_ir_module(cu, str(f), map_location, _extra_files)
RuntimeError: required keyword attribute 'chunks' is undefined
2022-06-14 10:58:09,677 - mmdeploy - ERROR - visualize torchscript model failed
```